Doc/library/asyncio-eventloop.rst

```diff
@@ -52,17 +53,19 @@ an event loop:
    :func:`get_running_loop` function is preferred to :func:`get_event_loop`
    in coroutines and callbacks.
 
-   Consider also using the :func:`asyncio.run` function instead of using
-   lower level functions to manually create and close an event loop.
+   As noted above, consider using the higher-level :func:`asyncio.run` function,
+   instead of using these lower level functions to manually create and close an
+   event loop.
 
    .. deprecated:: 3.10
-      Deprecation warning is emitted if there is no running event loop.
-      In future Python releases, this function will be an alias of
-      :func:`get_running_loop`.
+      Emits a deprecation warning if there is no running event loop.
+      In future Python releases, this function will become an alias of
+      :func:`get_running_loop` and will accordingly raise a
+      :exc:`RuntimeError` if there is no running event loop.
```

Comment:
Unfortunately, while for a long time that was indeed the plan, we are considering undeprecating it, based on feedback received from users who rely on this (see https://github.com/python/cpython/issues/93453). Maybe all we need to do, while this is as yet undecided, is to say "may" instead of "will"?

Reply:
Yep, reading the discussion, I think a switch to "may" would suffice.

Would it be good/useful also to link to that ongoing discussion? Or is the philosophy of the docs to avoid that kind of contemporaneity?

Reply:
I would skip the link, it will just confuse folks. We'll update these docs once we are sure.

Hopefully this PR can be backported to 3.11 and maybe 3.10.